docs/index.html

```diff
@@ -371,340 +337,685 @@ <h3>RP Meta-data</h3>
+            </ul>
 
-        <p>
-            The <a href="#rp">RP</a> MUST validate the <code>&lt;SIOP Response&gt;</code> as described in the
-            Self-Issued ID Token Validation section in [[!OIDC.Core]]. This includes:
-        </p>
+            <p class="note">
+                See <a href="#md-device-binding">Device Binding</a> for more details on the usage of
+                <code>verification_uri</code>, <code>device_code</code>, <code>expires_in</code> and
+                <code>interval</code>.
+            </p>
 
-        <ul>
-            <li>
-                Optionally decrypting the JWE to obtain the JWS which contains the <code>id_token</code>.
-            </li>
-            <li>
-                Verifying that the <code>id_token</code> was signed by the key specified in the <code>sub_jwk</code>
-                claim.
-            </li>
-        </ul>
+        </section>
 
-        <p>
-            Additionally, the <a href="#rp">RP</a> MUST validate the <code>id_token</code> against the
-            <a href="#siop">SIOP's</a> <a href="#did-document">DID Document</a> as follows:
-        </p>
+        <section>
+            <h3 id="md-siop-request-validation">Request Validation (Multi-device Flow)</h3>
 
-        <ul>
-            <li>
-                If no <code>did_doc</code> is present, resolve the <a href="#did-document">DID Document</a> from the
-                <a href="#siop">SIOP's</a> <a href="#did">DID</a> specified in the <code>did</code> claim.
-            </li>
-            <li>
-                If <code>did_doc</code> is present, ensure this is a viable channel to exchange the
-                <a href="#siop">SIOP's</a> <a href="#did-document">DID Document</a> according to the applicable DID
-                method.
-            </li>
-            <li>
-                Determine the verification method from the <a href="#siop">SIOP's</a> <a href="#did-document">DID
-                Document</a> that matches the <code>kid</code> of the <code>sub_jwk</code> claim in the
-                <code>id_token</code>.
-            </li>
-            <li>
-                Verify the <code>id_token</code> according to the verification method above. This step depends on the
-                verification method in the authentication section in the <a href="#did-document">DID Document</a> and
-                is out-of-scope of this specification.
-            </li>
-        </ul>
+            <p>
+                The <a href="#siop">SIOP</a> MUST follow the rules in
+                <a href="#sd-siop-request-validation">Request Validation (Regular Flow)</a> to validate
+                the <code>&lt;SIOP DID Request&gt;</code>.
+            </p>
+
+            <p>
+                Additionally, the following validation rules MUST be applied:
+            </p>
+
+            <ul>
+                <li>
+                    If no <code>verification_uri</code> is present, then continue with the Regular-device Flow.
```

Comment:
I'm not clear on what it means to process a Multi-Device Flow request where verification_code is not present -- shouldn't this be an error? I thnk this first bullet point is really about how a client can tell whether a request is using the Multi-Device flow or not, and it's odd because it's sitting in a section called "Request Validation (Multi-device Flow)".

Reply:
Normally, this could be done by defining a dedicated grant type in the request, e.g., `grant_type = urn:dif:params:oauth:grant-type:siop_multi_device`. Do you think it is reasonable to introduce that? 

Otherwise, we would need to guess what the intention of the requester was. For example, if any of the REQUIRED parameters are missing, I would consider this as an error, but if none of them are were provided, then the regular flow was intended. 

@jmandel Do you have a suggestion or preference?

Reply:
I think the proposed wire protocol here works (i.e., `verification_uri` present implies this is a multi-device request). I'd just suggest updating the langauge so that "3.3.2 Request Validation (Multi-device Flow)" is renamed something like "Determining which flow to follow".

That way it doen't seem like one branch of the multi-device flow is to perform a single-device flow.